Description: Use $HOME/Downloads as download path, as that's all that the
 apparmor profile supports.  Not forwarded as this is specific to Ubuntu,
 and specifically edits config.h...
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: no

Index: vimprobable2-1.4.2/main.c
===================================================================
--- vimprobable2-1.4.2.orig/main.c
+++ vimprobable2-1.4.2/main.c
@@ -3154,7 +3154,7 @@ main(int argc, char *argv[]) {
     else
         config->config_base = g_strdup_printf("%s/.config/", getenv("HOME"));
 
-    sprintf(downloads_path, "%s", getenv("HOME"));
+    sprintf(downloads_path, "%s/Downloads", getenv("HOME"));
 
     if (cfile)
         config->configfile = g_strdup(cfile);
